CompuCell3D/core/post_install_tasks/CMakeLists.txt

```diff
@@ -19,10 +19,8 @@ set(cc3d_py_source_dir ${CMAKE_SOURCE_DIR}/../cc3d)
 
 message("THIS IS cc3d_py_source_dir" ${cc3d_py_source_dir})
 
-if(INSTALL_PLAYER)
-#INSTALL(DIRECTORY  "@CMAKE_SOURCE_DIR@/${cc3d_py_dir}/CompuCellSetup" DESTINATION ${cc3d_py_dir_target}
+if(BUILD_PYINTERFACE)
```

Comment:
Not sure about this, since `BUILD_PYINTERFACE` also includes the Python API, which would probably be the intended user interface for the core package by itself.